improver_tests/wxcode/wxcode/test_utilities.py

```diff
@@ -367,50 +371,43 @@ def test_wxcode_updated_on_latlon(self):
 class Test_interrogate_decision_tree(IrisTest):
     """Test the function for generating extended help."""
```

Comment:
Is this class now empty? Can it be deleted?

Reply:
No, it's still used by the --check-tree option. If you run the --check-tree option for a tree that passes all the checks it returns the list of required inputs for that tree.

Reply:
I don't think it is. The test function `test_check_tree` and `test_check_tree_invalid_key` are not members of that class, or any other class. Looking at the `test_utilities.py` file, the `Test_interrogate_decision_tree` class has no members. I've tried deleting it and the same number of tests run, and pass.

Reply:
Sorry, I completely missed your point. Deleted.